src/annotator/config/index.js

```diff
@@ -8,9 +8,10 @@ import { urlFromLinkTag } from './url-from-link-tag';
  *
  * @typedef {import('./settings').SettingsGetters} SettingsGetters
  *
+ * @typedef {(settings: SettingsGetters, name: string) => any} ValueGetter
```

Comment:
A small stylistic note: When there are a group of single-line typedefs I tend to group them together without blank lines in between, just to let a little more useful code fit on a screen.

Reply:
I'll do that!